packages/framework/assets/js/common/validation/customizeFpValidator.js

```diff
@@ -17,7 +17,7 @@ FpJsFormValidator.preparePrototype = function (prototype, name) {
 
     if (typeof prototype.children === 'object') {
         for (let childName in prototype.children) {
-            prototype[childName] = this.preparePrototype(prototype.children[childName], name);
+            prototype.children[childName] = this.preparePrototype(prototype.children[childName], name);
```

Comment:
```suggestion
            prototype.children[childName] = this.preparePrototype(childName, name);
```

I am really not a JS expert, but would not be this one more obvious?

Reply:
childName is the key, so sadly no